build.gradle

```diff
@@ -37,6 +37,7 @@ subprojects {
 golang {
     packagePath = 'github.com/apache/incubator-openwhisk-cli' as String
     buildTags = (rootProject.findProperty('goTags')?:'').split(',')
+    goVersion = '1.9.3'
```

Comment:
current go version used to build the published cli is 1.9
https://github.com/apache/incubator-openwhisk-cli/blob/701107f9d73dc455d67e3409a1a9f22235d37bb0/.travis.yml#L25

Reply:
yeap, but specifying the subversion should be fine as long as the major version doesn't change.

Reply:
For some reason gogradle cannot download the `1.9.0` on my mac, `1.9.1` onwards are available though

Reply:
in general, i think we should be consistent when pinning the version of a tool.

Reply:
can you say which patch version (e.g. 1.9.x) is used by travis?

Reply:
since travis allows a `1.9.x` specification to allow for the latest patch level, i believe the current travis setting of `1.9` will pull the `1.9.0` version.  this is confirmed by the travis output
```
$ go version
go version go1.9 linux/amd64
```